monai/apps/pathology/transforms/spatial/array.py

```diff
@@ -50,17 +53,47 @@ def __init__(
         else:
             self.patch_size = patch_size
 
-    def __call__(self, image: torch.Tensor) -> torch.Tensor:
+    def __call__(self, image: Union[np.ndarray, torch.Tensor]) -> Union[np.ndarray, torch.Tensor]:
```

Comment:
Please use `NdarrayOrTensor` instead, same in all the other places of this PR.
Refer to: https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/croppad/array.py#L72

Thanks.